src/typescript-worker/project-cache.ts

```diff
@@ -0,0 +1,15 @@
+import { TypeScriptBuilder } from "./typescript";
+import {CachingCdn} from './caching-cdn.js';
+import { ImportMapResolver } from "./import-map-resolver";
+
+
+export const builderCache = new Map();
+
+export function getBuilder(projectId: string, cdn: CachingCdn, moduleResolver: ImportMapResolver) {
```

Comment:
Related to other comment -- since we know that there's a 1-1 relationship between each `<playground-project>` and its dedicated web worker, I think we don't actually need this cache. We can just make one module-scoped TypeScript builder :D